api/v2/views/base.py

```diff
@@ -24,8 +24,9 @@ def unresolved_requests_only(fn):
     @wraps(fn)
     def wrapper(self, request, *args, **kwargs):
         instance = self.get_object()
+        user_can_act = request.user.is_staff or request.user.is_superuser or CloudAdministrator.objects.filter(user=request.user.id)
```

Comment:
Correct! That example probably has an error in it. What it probably should show is.
```python
some_queryset = Entry.objects.filter(pk=123)
if some_queryset.exists():
    entry = some_query.first()
    print("Entry contained in queryset")```

Or it should show the example you listed above.